src/org/labkey/mobileappstudy/MobileAppStudyModule.java

```diff
@@ -41,7 +41,7 @@
 
 public class MobileAppStudyModule extends DefaultModule
```

Comment:
Might as well change the class name. This means updating module.properties and the core.Modules update query as well.

Reply:
Updated all package and non-schema class names to `Response*`. I did not change any JavaScript `LABKEY.mobilaAppStudy` names.